### PR TITLE
Example 8.14

### LINKED ISSUE #14: Formalise covariant functoriality (Section 8.2 of RS17 paper)
- [x] Example 8.14 (#29)
- [ ] Lemma 8.15

### PR DESCRIPTION
**Pull request by TashiWalde**
partially address #14

### CODE REVIEW

**Review by emilyriehl — COMMENTED** [review]
Wow! I'm surprised this is provable by refl. Well observed.

Some minor cosmetic suggestions before we merge:

1. The title is slightly better as "RS17, Example 8.14" because other things in the library reference other sources.

2. The line where you've declared the type of your term is too long. (For me, in vsCode, there is a light grey vertical line that tells me where the character limit is. You can also eyeball with the rest of the file.) It's not so clear to me where is best to make the line break. One option is

```rzk
: covariant-transport 
      A x y f (hom A a) (is-covariant-representable-is-segal A is-segal-A a) e = 
  comp-is-segal A is-segal-A a x y e f
```

3. Finally we should consider the term name. @fizruk or @fredrik-bakke what would you call this?

**Comment by fredrik-bakke** [comment]
> partially resolves #14

You have to be careful with words like "fixes" and "resolves" in GitHub. With the way your comment is phrased, GitHub will mark the issue as resolved when this PR is merged, so I suggest a different phrasing like "Addresses item 1 in #14.".

File: src/simplicial-hott/08-covariant.rzk.md [hunk 1]
```diff
@@ -768,6 +768,23 @@ along an arrow f : hom A x y to give a term in C y.
     first (center-contraction (dhom-from A x y f C u) (is-covariant-C x y f u))
 ```
 
+Example: If A is a Segal type and a : A, the family C x := hom A a x is covariant.
+Transport an e : C x, along an arrow f : hom A x y just yields composition of f with e.
```
Comment (fredrik-bakke) [inline src/simplicial-hott/08-covariant.rzk.md:772]:
```suggestion
Example: If `A` is a Segal type and `a : A`, the family `C x := hom A a x` is covariant.
Transport an `e : C x`, along an arrow `f : hom A x y` just yields composition of `f` with `e`.
```

**Comment by fredrik-bakke** [comment]
How about `compute-covariant-transport-of-hom-family-is-segal` for this definition? It reads like a computation principle to me, and as always, we append the assumption that A is Segal. Another alternative is `eq-comp-covariant-transport-of-hom-family-is-segal`, where we prepend the concluding type as always, using "comp" instead of "composition" to distinguish it from composition witnesses.

**Review by fizruk — APPROVED** [review]
I think, we can merge this now.

**Pull request merged**